packages/adblocker-extended-selectors/src/eval.ts

```diff
@@ -211,6 +255,9 @@ export function matches(element: Element, selector: AST): boolean {
       return (
         selector.argument !== undefined && matchCSSProperty(element, selector.argument, '::before')
       );
+    } else if (selector.name === 'xpath') {
+      const matches = handleXPathSelector(element, selector.argument);
+      return matches.includes(element);
```

Comment:
If we can follow https://github.com/ghostery/adblocker/pull/4933/files#r2180150577, we don't need this changes.